media-tv/kodi/kodi-21.2.ebuild

```diff
@@ -0,0 +1,545 @@
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	|| ( gbm wayland X )
+	?? ( mariadb mysql )
+	bluray? ( udf )
+	gbm? ( udev )
+	soc? ( system-ffmpeg )
+	udev? ( !libusb )
+	vdpau? ( X !gles !gbm )
+	zeroconf? ( dbus )
+"
+RESTRICT="!test? ( test )"
+
+# dev-libs/libcec[-cubox] bug #818262
+COMMON_DEPEND="
+	>=dev-libs/flatbuffers-23.3.3:=
+	>=dev-libs/lzo-2.04:2
+	media-libs/giflib:=
+	>=media-libs/libjpeg-turbo-2.0.4:=
+	>=media-libs/libpng-1.6.26:0=
+	wayland? (
+		dev-cpp/waylandpp:=
+	)
+"
+COMMON_TARGET_DEPEND="${PYTHON_DEPS}
+	>=net-misc/curl-7.68.0[http2]
+	>=sys-libs/zlib-1.2.11
+	dev-db/sqlite:3
+	dev-libs/crossguid
+	>=dev-libs/fribidi-1.0.5
+	>=dev-libs/libcdio-2.1.0:=[cxx]
+	>=dev-libs/libfmt-6.1.2:=
+	dev-libs/libfstrcmp
+	dev-libs/libpcre[cxx]
+	>=dev-libs/openssl-1.1.1k:0=
+	>=dev-libs/spdlog-1.5.0:=
+	dev-libs/tinyxml[stl]
+	dev-libs/tinyxml2:=
+	media-fonts/roboto
+	media-libs/libglvnd[X?]
+	>=media-libs/freetype-2.10.1
+	media-libs/harfbuzz:=
+	>=media-libs/libass-0.15.0:=
+	media-libs/mesa[egl(+),gbm(+)?,wayland?,X?]
+	media-libs/taglib:=
+	sci-libs/kissfft
+	virtual/libiconv
+	virtual/ttf-fonts
+	x11-libs/libdrm
+	$(python_gen_cond_dep '
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/pycryptodome[${PYTHON_USEDEP}]
+	')
+	airplay? (
+		>=app-pda/libplist-2.0.0:=
+		net-libs/shairplay
+	)
+	alsa? (
+		>=media-libs/alsa-lib-1.1.4.1
+	)
+	bluetooth? (
+		net-wireless/bluez:=
+	)
+	bluray? (
+		>=media-libs/libbluray-1.1.2:=
+	)
+	caps? (
+		sys-libs/libcap
+	)
+	cec? (
+		>=dev-libs/libcec-4.0[-cubox]
+	)
+	dbus? (
+		sys-apps/dbus
+	)
+	gbm? (
+		>=dev-libs/libinput-1.10.5:=
+		media-libs/libdisplay-info
+		x11-libs/libxkbcommon
+	)
+	gles? (
+		|| (
+			>=media-libs/mesa-24.1.0_rc1[opengl]
+			<media-libs/mesa-24.1.0_rc1[gles2]
+		)
+	)
+	!gles? (
+		media-libs/glu
+	)
+	lcms? (
+		>=media-libs/lcms-2.10:2
+	)
+	libusb? (
+		virtual/libusb:1
+	)
+	lirc? (
+		app-misc/lirc
+	)
+	mariadb? (
+		dev-db/mariadb-connector-c:=
+	)
+	mysql? (
+		dev-db/mysql-connector-c:=
+	)
+	nfs? (
+		>=net-fs/libnfs-2.0.0:=
+	)
+	pipewire? (
+		>=media-video/pipewire-0.3.50:=
+	)
+	pulseaudio? (
+		media-libs/libpulse
+	)
+	samba? (
+		>=net-fs/samba-3.4.6[smbclient(+)]
+	)
+	system-ffmpeg? (
+		=media-video/ffmpeg-6*:=[encode,soc(-)?,postproc,vaapi?,vdpau?,X?]
+	)
+	!system-ffmpeg? (
+		app-arch/bzip2
+		app-arch/xz-utils
+		media-libs/dav1d:=
+		net-libs/gnutls:=
+	)
+	udf? (
+		>=dev-libs/libudfread-1.0.0
+	)
+	udev? (
+		virtual/libudev:=
+	)
+	vaapi? (
+		media-libs/libva:=[wayland?,X?]
+	)
+	vdpau? (
+		|| (
+			>=x11-libs/libvdpau-1.1
+			>=x11-drivers/nvidia-drivers-180.51
+		)
+	)
+	wayland? (
+		>=x11-libs/libxkbcommon-0.4.1[wayland]
+	)
+	webserver? (
+		>=net-libs/libmicrohttpd-0.9.77:=[messages(+)]
```

Comment:
No versions of net-libs/libmicrohttpd have the messages flag anymore; the functionality is enabled by default. We can depend on net-libs/libmicrohttpd without any specific flags.